Cubical/Categories/Displayed/Functor/More.agda

```diff
@@ -94,3 +94,47 @@ module _
 
       GF-hom : GF-hom-ty GF-ob
       GF-hom = _ , GF-hom≡FF-hom
+
+Functorⱽ : {C : Category ℓC ℓC'}
+  → Categoryᴰ C ℓCᴰ ℓCᴰ' → Categoryᴰ C ℓDᴰ ℓDᴰ'
+  → Type _
+Functorⱽ = Functorᴰ Id
+
+module _ {C : Category ℓC ℓC'} {Cᴰ : Categoryᴰ C ℓCᴰ ℓCᴰ'}
+  {Dᴰ : Categoryᴰ C ℓDᴰ ℓDᴰ'} {Eᴰ : Categoryᴰ C ℓEᴰ ℓEᴰ'}
+  (Gᴰ : Functorⱽ Dᴰ Eᴰ) (Fᴰ : Functorⱽ Cᴰ Dᴰ)
+  where
+
+  funcCompⱽ : Functorⱽ Cᴰ Eᴰ
+  funcCompⱽ = reindF' _ Eq.refl Eq.refl (Gᴰ ∘Fᴰ Fᴰ)
```

Comment:
I'm not sure which thing you are asking about: 
1. We want this definition `funcCompⱽ` just because it is useful, e.g., composing with the projection functor for reindexing we usually need to get rid of the composition with the identity functor in the base.
2. `reindF'` just has better definitional behavior than `reindF` when the proofs are `Eq.refl` since `reindF` does a reindex/transport. `reindF' _ Eq.refl Eq.refl` is instead like getting the displayed functor to type check by eta expansion but less tedious.

Reply:
I was wondering more of the second. As in, which reindex exactly does this do away with. I'll play around with it later.